private/rules/java_export.bzl

```diff
@@ -12,6 +12,7 @@ def java_export(
         deploy_env = [],
         excluded_workspaces = {name: None for name in DEFAULT_EXCLUDED_WORKSPACES},
         exclusions = {},
+        implementation_deps = [],
```

Comment:
Could we call this "runtime_deps"?

Reply:
Opting to continue calling this implementation_deps. They are technically different.

`implementation_deps` are "compile" time for the library itself but "runtime" for the exported pom. It is really only relevant to pom generation.